.circleci/config.yml

```diff
@@ -93,7 +97,7 @@ jobs:
 
   test_app:
     <<: *android_config
-    parallelism: 83
+    parallelism: 50
```

Comment:
I think pushing `parellelism` TO THE MAX was causing our parallel jobs to get queued.